src/tests/conditions.test.js

```diff
@@ -423,3 +423,59 @@ describe('Conditional attributes updated', () => {
     expect(fields[1].visibilityCondition).toEqual(expect.any(Function));
   });
 });
+
```

Comment:
nit: A better PR title (commit) name:

```diff
- fix(conditionals): Certain conditions were failing due to lack of context #65
- fix(conditionals): Add missing field context to conditions #65
```

Reply:
But to be honest I'm not sure 🤔 We don't have any guidelines either. Thoughts?

Reply:
Can be a mix of both. Problem and solution.

```
fix(conditionals): Certain conditions were failing. Add missing field context to evaluation
```

Reply:
Love it, will use for merge commit.

Reply:
but up to you!

Reply:
(oh github failed on me for the updates, perfect then)